include/aspect/material_model/steinberger.h

```diff
@@ -252,6 +252,13 @@ namespace aspect
 
 
       private:
+        /**
+         * Returns the enthalpy derivatives for the evaluate function and
+         * postprocessors.
+         */
```

Comment:
Can you add an explanation somewhere which one is the temperature and which one is the pressure derivative?